### PR TITLE
[dev] updated gitignore with additional extensions to ignore

### PR DESCRIPTION
**Pull request by eapostol**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -47,3 +47,4 @@ jspm_packages
 .node_repl_history
 
 ./.idea/sonarlint/
+!/.idea/conflict-resolution.iml
```
Comment (eapostol-2u) [inline .gitignore:50]: This file should be ignored, the exclamation mark negates this, so let's fix it.

**Review by eapostol-2u — DISMISSED** [review]
Please remove the ! from the conflict-resolution.iml setting in the .gitignore file.

File: .gitignore (hunk 1 above)
Comment (eapostol) [reply .gitignore]: I have added it in the next commit, this should commit should be resolved.

**Review by eapostol-2u — APPROVED** [review]
the exclamation mark was removed.

**Pull request merged**